internal/middleware/account_test.go

```diff
@@ -21,9 +20,9 @@ func TestAccountMiddleware(t *testing.T) {
 		ctx := stubs.WithAccountDaoOne(context.Background())
 		ctx = identity.WithTenant(t, ctx)
 
-		req, err := http.NewRequestWithContext(ctx, "GET", "/api/provisioning/v1/pubkeys", nil)
+		req, err := http.NewRequestWithContext(ctx, "GET", "/test", nil)
```

Comment:
If it is an account test I would change it `/account_test` but not so important 😄

Reply:
well I wanted to be clear that the url has no meaning, `/account_test` could still imply we have some endpoint set up for testing account, so I'd keep it general to really underline it does nothing at all.

Reply:
Unless we want `/useless_path`, but I thought that being bit on the nose xD